webapp/channels/src/utils/utils.tsx

```diff
@@ -212,6 +212,31 @@ const removeQuerystringOrHash = (extin: string): string => {
 };
 
 export const getFileType = (extin: string): typeof FileTypes[keyof typeof FileTypes] => {
```

Comment:
Lots of good testing already, but just to double check: is there a pure unit test we can update for `getFileType`?

Reply:
done, added new tests.